Snippets/Sqs/Sqs_All/SqsTransportConfigurationExtensions.cs

```diff
@@ -6,9 +6,9 @@
 
     public static class SqsTransportConfigurationExtensions
     {
-        internal const string RegionEnvironmentVariableName = "NServiceBus.AmazonSQS.Region";
-        const string S3BucketEnvironmentVariableName = "NServiceBus.AmazonSQS.S3Bucket";
-        const string NativeDeferralEnvironmentVariableName = "NServiceBus.AmazonSQS.NativeDeferral";
+        internal const string RegionEnvironmentVariableName = "NServiceBus_AmazonSQS_Region";
+        const string S3BucketEnvironmentVariableName = "NServiceBus_AmazonSQS_S3Bucket";
+        const string NativeDeferralEnvironmentVariableName = "NServiceBus_AmazonSQS_NativeDeferral";
```

Comment:
Should we ditch this because the API never really had an effect?

Reply:
👍